### ISSUE TITLE
there is apparently no style corresponding to the language name or the memory limit text in the interactions area

### ISSUE THREAD

**Issue by HeladoDeBrownie** [racket/drracket#319]
in the interactions area, there's text at the top that says something like

> Language: racket, with debugging; memory limit: 128 MB.

and i've tried setting every available color and style in my color scheme (reported by running `racket -W info@color-scheme -l drracket`), but none of them changes the green text used for `racket, with debugging` and `128 MB`. the color contrasts poorly with the background color i'm using, which is `#( 90  84 117)`.

**Comment by gfbee** [comment]
Yes, it's hard-coded (programatically there are parameters here: https://docs.racket-lang.org/tools/drracket_rep.html?q=delta#%28def._%28%28lib._drracket%2Ftool-lib..rkt%29._drracket~3arep~3aget-welcome-delta%29%29 ). It would be great to have the various sort-of constants inside drracket turned into preferences.

**Comment by HeladoDeBrownie** [comment]
thanks for pointing that out! at a glance i'm guessing these wouldn't easily be changed by a color scheme alone in their current state? (most importantly the colors would need to change on the spot when switching between color schemes in the preferences.)

**Comment by rfindler** [comment]
The commit 79d8442 adjusts the blue mentioned above so that it is sensitive to the light/dark boolean of a color scheme, but doesn't actually let the them control the precise shade. Let me know if that's enough or not.

**Comment by HeladoDeBrownie** [comment]
where can i find instructions on how to run the repository version of drracket? alternatively, if someone can post a screenshot using the color scheme from [this package](https://github.com/HeladoDeBrownie/DrRacket-Theme-Fairy-Floss) and showing the affected text, i would appreciate it!

**Comment by rfindler** [comment]
Here's how it looks for me:

<img width="509" alt="Screen Shot 2019-12-25 at 8 46 20 PM" src="https://user-images.githubusercontent.com/399024/71454720-a4dff480-2757-11ea-992d-b0dd71e67e4e.png">

**Comment by HeladoDeBrownie** [comment]
that contrast is acceptable! thank you so much!

**Comment by rfindler** [comment]
As for using DrRacket, on linux or mac os, I think the simplest thing is to just clone http://github.com/racket/racket and run "make" from the top-level directory (this builds in place; it can take a little while but it makes sure you get pre-release versions of everything that is all compatible with itself (unless someone has made a mistake and that's rare and fixed quickly). If you want to edit things and make pull requests, that's also the first step on the recommended path, too. On Windows, that's the best option too, I think (but maybe others know better), but getting a C compiler set up properly is more complicated.

**Comment by HeladoDeBrownie** [comment]
okay! i might try that if i manage to set some time aside to contribute something myself. thanks!

**Comment by rfindler** [comment]
Also, if you put the colorscheme on https://pkgs.racket-lang.org/ it will be easier for people to install it. It looks really nice!

**Comment by HeladoDeBrownie** [comment]
i've stalled working on it but i do plan to look into posting it there if i finish it (incidentally i didn't design the original theme, i'm just porting it to drracket, but thanks!)

**Comment by HeladoDeBrownie** [comment]
@rfindler it's on there now under the name of `fairyfloss` (incidentally i tried adding a 1.0.0 link for it in addition to the default one but got an error message saying only `Save failed.`)
